Source/ACE/Entity/Door.cs

```diff
@@ -283,7 +254,7 @@ public override void OnUse(ObjectGuid playerId)
         private void Open(ObjectGuid opener = new ObjectGuid())
         {
             ActionChain chain = new ActionChain();
-            CurrentLandblock.ChainOnObject(chain, Guid, (WorldObject wo) =>
+            chain.AddAction(this, () =>
```

Comment:
Open is now called exclusively from an ActionChain on the Door, this chain is redundant.